guides/traversing-pages.rst

```diff
@@ -81,6 +81,37 @@ that you can make same old traversing on such elements as well:
 
     // find some field INSIDE form with class="register"
     $field = $registerForm->findField('Email');
+    
+Webassert
+~~~~~~~~~
+
+The ``\Behat\Mink\WebAssert`` class provides a set of assertion. There are assertions
+about the address of the page, the cookies, the status code, the response headers,
+the content of the page, the page elements... An assertion can improve code readability
+and avoid having a fatal error in method chaining.
+
+For example, instead of:
+
+.. code-block:: php
+
+    $registerForm = $page->find('css', 'form.register');
+
+    if (null === $registerForm) {
+        throw new \Exception('The element is not found');
+    }
+
+    // find some field INSIDE form with class="register"
+    $field = $registerForm->findField('Email');
+
+you can do:
+
+.. code-block:: php
+
+    field = $mink->assertSession()->fieldExists(
```

Comment:
Are you sure, that `assertSession` should get any session or it's name as a parameter?